### PR TITLE
Fix the presubmit unit test.

### PR DESCRIPTION
**Pull request by fbeevikm**
Update the docker image to developer-tools-light
Upgrade golangci-lint to v1.47.3

### CODE REVIEW

File: build/presubmit_int.yaml [hunk 1]
```diff
@@ -12,9 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Terraform version used is 0.14.8.
+# Terraform version used is 1.2.9.
 substitutions:
-  _CFT_TOOLS_CONTAINER: "gcr.io/cloud-foundation-cicd/cft/developer-tools@sha256:8b0d85766044b0e4701729f9e70ee78dcecdb8688bd86915eb2a1cb00580ba1e"
+  _CFT_TOOLS_CONTAINER: "gcr.io/cloud-foundation-cicd/cft/developer-tools-light@sha256:e4ef6a7cb46ca110b97eb5fb7134fb9f3720b98560436792a4199c77e739dca0"
```
Comment (yeweidaniel) [inline build/presubmit_int.yaml:17]: Out of curiosity, how did you find which image to use and what's the difference in developer-tools-light?

File: templates/tfengine/components/terraform/main/main.tf [hunk 2]
```diff
@@ -40,7 +40,7 @@ terraform {
     google-beta = ">= 3.0"
     {{end -}}
     {{if not $hasKubernetes -}}
-    kubernetes  = "~> 1.0"
+    kubernetes  = "~> 2.10"
```
Comment (yeweidaniel) [inline templates/tfengine/components/terraform/main/main.tf:43]: Is this because the new Terraform version no longer supports 1.0 provider?

**Review by yeweidaniel — APPROVED** [review]

File: build/presubmit_int.yaml (hunk 1 above)
Comment (fbeevikm) [reply build/presubmit_int.yaml:17]: I had updated the cicd to use this image for tf-apply and it worked fine. This one has terraform and cloud sdk. Deveoper tools has other packages installed too. I don't think we are using that here.

File: templates/tfengine/components/terraform/main/main.tf (hunk 2 above)
Comment (fbeevikm) [reply templates/tfengine/components/terraform/main/main.tf:43]: Yes.

**Pull request merged**